spec/unit/comments_spec.rb

```diff
@@ -55,6 +55,17 @@
         expect(comments.second).to eq(yet_another_comment)
         expect(comments.last).to eq(another_comment)
       end
+
+      it "should return the correctly ordered comments" do
+        ActiveAdmin::Application.inheritable_setting(:comment_order, 'created_at DESC')
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.